src/Config.js

```diff
@@ -15,14 +16,16 @@ class Config {
 
   _config;
 
-  constructor() {
```

Comment:
Since we are currently using Config only in one place (the manager) and it is being changed to be invoked with a constructor parameter, we should remove the Singleton plumbing below (ex. `if (Config.instance)` etc.